README.md

```diff
@@ -4,13 +4,15 @@ This repository serves as a check-in for attendees of the **Git Workshop for Mod
 
 ---
 ## Attendees
+=======

```

Comment:
We believe we can remove this line.
```suggestion
```

Reply:
OH WOOPS I MISSED THAT MY B

Reply:
Also don't forget to resolve the new merge conflicts!